Classes/Extensions/DDDispatchQueueLogFormatter.h

```diff
@@ -24,6 +24,11 @@
 #import "DDLog.h"
 
 
+typedef NS_OPTIONS(NSUInteger, DDDispatchQueueLogFormatterOptions) {
+    DDDispatchQueueLogFormatterShareable = 1 << 0,
+};
```

Comment:
It is a bit flag, there is 2 options already: flag ON or OFF
Though on the second though I think it'd be more intuitive if default behaviour corresponds to `0` options mask.